frontend/src/components/page-events/event-processing.js

```diff
@@ -41,6 +41,7 @@ function processEvents(e) {
         event.LocationOverride?.LocationName ||
         event.EventTemplate?.Location.LocationName ||
         "",
+      googleMapsLink: event?.EventTemplate?.Location?.GoogleMapsLink,
```

Comment:
this should prioritize the event's location first: 

`event.LocationOverride?.GoogleMapsLink || event.EventTemplate?.Location?.GoogleMapsLink`